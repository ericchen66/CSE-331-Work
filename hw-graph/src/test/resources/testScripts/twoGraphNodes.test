# Tests adding nodes in two separate graphs

# Creates two graphs
CreateGraph graph1
CreateGraph graph2

# Adds one node to both graphs
AddNode graph1 node1
AddNode graph2 node2

# Lists nodes of each graph
ListNodes graph1
ListNodes graph2