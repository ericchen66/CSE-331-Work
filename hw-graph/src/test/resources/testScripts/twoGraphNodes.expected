# Tests adding nodes in two separate graphs

# Creates two graphs
created graph graph1
created graph graph2

# Adds one node to both graphs
added node node1 to graph1
added node node2 to graph2

# Lists nodes of each graph
graph1 contains: node1
graph2 contains: node2